### PR TITLE
feat: rename action endpoints

### PR DESCRIPTION
**Pull request by rauno56**
`GET /actions/types/RenameAttribute/:id`

```json
{
  "actionName": "Rename network attributes",
  "signals":
  [
    "LOGS",
    "METRICS",
    "TRACES"
  ],
  "renames":
  {
    "net.host.ip": "network.local.address",
    "net.local.address": "network.local.address",
    "net.peer.address": "network.peer.address",
    "net.peer.ip": "network.peer.address"
  }
}
```

Example rename action(apply with `kubectl` for testing if needed):

```yaml
apiVersion: actions.odigos.io/v1alpha1
kind: RenameAttribute
metadata:
  name: ra-9sz8q
  namespace: odigos-system
spec:
  actionName: Rename network attributes
  renames:
    net.peer.address: network.peer.address
    net.local.address: network.local.address
    net.peer.ip: network.peer.address
    net.host.ip: network.local.address
  signals:
  - LOGS
  - METRICS
  - TRACES

```

### CODE REVIEW

**Review by blumamir — APPROVED** [review]
Thank you @rauno56 

What about adding this action to the odigos docs? is it something you plan for a followup PR?

**Comment by rauno56** [comment]
Writing the docs right right now actually.

**Comment by rauno56** [comment]
Added a fix for e2e tests helm charts for the time being.

File: docs/pipeline/actions/rename-attribute.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Rename Attribute"
+sidebarTitle: "Rename Attribute"
+---
+
+The "Rename Attribute" Odigos Action can be used to rename attributes from logs, metrics, and traces. Different instrumentations might use different attribute names for similar information. This action let's you to consolidate the names across your cluster.
+
+### Mechanism
+
+The action will remove attributes from the following contexts:
```
Comment (blumamir) [inline docs/pipeline/actions/rename-attribute.mdx:10]:
```suggestion
The action will rename attributes from the following contexts:
```

File: docs/pipeline/actions/rename-attribute.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The basic example below shows how to rename the deprecated `net.peer.address` attribute to stable `network.peer.address` in all telemetry signals.
+
+Create a file named `rename-net.action.yaml` with the following content:
+
+```yaml
+apiVersion: actions.odigos.io/v1alpha1
+kind: RenameAttribute
+metadata:
+  name: ra-9sz8q
```
Comment (blumamir) [inline docs/pipeline/actions/rename-attribute.mdx:31]: I think it's better to use a meaningful name here, which will also be aligned with other actions and better practice for the users if they create it manually

**Review by blumamir — APPROVED** [review]

File: docs/pipeline/actions/rename-attribute.mdx (hunk 2 above)
Comment (rauno56) [reply docs/pipeline/actions/rename-attribute.mdx]: True that. Thanks!

**Pull request merged**